auth0_flutter/EXAMPLES.md

```
@@ -1419,6 +1414,58 @@ final credentials = await auth0Web.customTokenExchange(
 
 > 💡 For more information, see the [Custom Token Exchange documentation](https://auth0.com/docs/authenticate/custom-token-exchange) and [RFC 8693](https://tools.ietf.org/html/rfc8693).
 
+#### Delegation and impersonation (actor token)
+
+For delegation or impersonation scenarios, where one principal acts on behalf of another (for example, an AI agent acting on behalf of a user), provide an `actor` as an `ActorToken`, which bundles the token with its type so the pair can never be partially supplied.
+
+When present, the resulting ID token may contain an `act` claim (set server-side via an Auth0 Action using `api.authentication.setActor()`), exposed on the returned credentials' user profile via `.actor`.
+
+> **Note:** When an actor token is provided, Auth0 does **not** issue a refresh token regardless of whether `offline_access` is in the requested scopes, so `credentials.refreshToken` will be `null` in this flow.
+
```

markdownlint-cli2 (0.23.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)